package.json

```diff
@@ -575,5 +575,11 @@
     },
     "extensionDependencies": [
         "ms-vscode.azure-account"
-    ]
+    ],
+    "__metadata": {
```

Comment:
Where did this come from??

Reply:
Added automatically by the new test package.

Reply:
It's not in the Functions repo [though](https://github.com/microsoft/vscode-azurefunctions/blob/main/package.json#L1117). Did we explicitly remove it?

Reply:
Yeah, I'd prefer to remove it. It's puzzling why it's being added at all, I think we should raise it with the package authors.

Reply:
I removed it, but according to https://github.com/microsoft/vscode/issues/148975#issuecomment-1124930941 it's intentionally added and "can be used later for other features like updating etc.,"

Reply:
I think it makes sense for VSCode to update package.json in the installed extensions folder, but I don't think it should be doing it in source (like it currently is here).